README.md

```diff
@@ -1,26 +1,13 @@
-# BIIGLE Community Scripts
```

Comment:
You accidentally replaced the global readme of this repository. Please create a subdirectory (with your script and readme) and update the list in the global readme.